gui/src/theme_manager.cpp

```diff
@@ -0,0 +1,39 @@
+#include <QFile>
+#include <QIcon>
+
+#include <scopy/gui/theme_manager.hpp>
+
+using namespace scopy::gui;
+
+ThemeManager::ThemeManager()
+{
+	m_availableThemes = new QStringList();
+
+	m_availableThemes->append("default");
+	m_availableThemes->append("light");
+}
+
+ThemeManager& ThemeManager::getInstance()
+{
+	static ThemeManager INSTANCE;
+	return INSTANCE;
+}
+
+void ThemeManager::setApplication(QApplication* app) { m_app = app; }
+
+void ThemeManager::setCurrentTheme(const QString& theme)
+{
+	m_currentTheme = theme;
+
+	QFile file(":/stylesheets/" + theme + ".qss");
```

Comment:
We have this kind of read from file logic in a `utils` class. If that was not yet moved add a `TODO` here to use the utils function to avoid duplicate code.